### PR TITLE
fix(nms): upgrade axios and alarms

### PR DESCRIPTION
**Pull request by lucasgonze**
## Summary

Upgrade axios from 0.21.0 to 0.21.1 to pick up security-related patch

## Test Plan
Verified using unit tests:
* `cd ${MAGMA_ROOT}/feg/gateway/docker && ./build.py -c` 
* `cd ${MAGMA_ROOT}/orc8r/cloud/docker && ./build.py -t`

## Additional Information

Partial fix for https://github.com/ospoco/private-issue-tracker/issues/12

Signed-off-by: Lucas Gonze <lucas@gonze.com>

- [ ] This change is backwards-breaking

### CODE REVIEW

File: nms/packages/magmalte/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@babel/register": "^7.13.16",
     "@babel/runtime": "^7.14.0",
     "@date-io/moment": "1.3.13",
-    "@fbcnms/alarms": "^0.3.1",
+    "@fbcnms/alarms": "^0.1.0",
```
Comment (andreilee) [inline nms/packages/magmalte/package.json]: Why is this being downgraded?

**Review by andreilee — CHANGES_REQUESTED** [review]
`yarn.lock` needs to be updated as well

File: nms/packages/magmalte/package.json (hunk 1 above)
Comment (lucasgonze) [reply nms/packages/magmalte/package.json]: You caught a typo. Thank you. This should have been upgraded from 0.3.1 to 1.0.0.

**Review by lucasgonze — COMMENTED** [review]
Thanks for catching the alarms versioning issue. The most recent PR has alarms at 1.0.0 and axios at 0.21.2

**Comment by lucasgonze** [comment]
> `yarn.lock` needs to be updated as well

Done

**Comment by lucasgonze** [comment]
Stale. Creating a new PR with latest master.

**Pull request closed without merging**